.pre-commit-config.yaml

```
@@ -15,15 +15,16 @@ repos:
     rev: v1.1.334
     hooks:
       - id: pyright
-        files: plugins.*\/src.*\.py
+        files: (plugins.*\/src|sphinx_ext).*\.py
```

Was just digging into this a bit, there's a `types` filter in pre-commit: https://pre-commit.com/#filtering-files-with-types
So could in theory do `types: ['file', 'python']` instead of `.*\.py` at the end of the regex. That would avoid sym-links, but it also matches files with shebang:
> Files can also be matched by shebang. With types: python, an exe starting with #!/usr/bin/env python3 will also be matched.

Anyway, not saying we should do that, just thought it was interesting to know. Might be a little cleaner.

But here the regex is incorrect, it should be cleaned up for sure. There's no `/` after the plugins folder, so it would technically match something like `plugins-test/src-bad-file.py`. Should have the correct slashes:
```suggestion
        files: (plugins\/.*\/src|sphinx_ext)\/.*\.py
```